.github/workflows/chart-release.yaml

```diff
@@ -23,11 +22,21 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
       - name: Install Helm
         uses: azure/setup-helm@v1
         with:
           version: v3.4.1
-          
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.2.0
+
+      - name: Run chart-testing (lint)
+        run: ct lint --validate-maintainers=false --check-version-increment=false
```

Comment:
think the ct lint should be part of all PR pipelines. So `build-and-test.yaml` with the normal lint job. or you would only see a wrong value constellation after it's already in master.

Reply:
I've removed this job from chart-release pipeline and create `helm-lint` job in the `build-and-test` one. 

Now it's a separate job. Example: https://github.com/kloeckner-i/db-operator/actions/runs/

There are still a couple of changes in [chart-release.yaml](https://github.com/kloeckner-i/db-operator/blob/158c452dbce3b8ba3b65c0c7d8f8ab333154e86e/.github/workflows/chart-release.yaml) but those are formatting things.